kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/dsl/operations/SubscriptionDSL.kt

```diff
@@ -1,54 +1,34 @@
-package com.apurebase.kgraphql.schema.dsl
-
-import com.apurebase.kgraphql.Context
-import com.apurebase.kgraphql.schema.*
-import com.apurebase.kgraphql.schema.model.*
+package com.apurebase.kgraphql.schema.dsl.operations
+
+import AbstractOperationDSL
```

Comment:
This doesn't look right

Reply:
What do you mean by that?

Reply:
found out what is wrong. This line should be removed and you are missing package definition in `AbstractOperationDSL`

Reply:
You are right. I have overseen that, thanks. (it's fixed)